### ISSUE TITLE
Not working for other locales than English an Spanish

### ISSUE THREAD

**Issue by olivergierke** [jpenren/thymeleaf-spring-data-dialect#2]
Trying to use the dialect for locales other than English and Spanish, the rendering fails with (in this case German):

````
java.util.MissingResourceException: Can't find bundle for base name thymeleaf-spring-data-dialect/FullPaginationDecorator, locale de_DE
	at java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:1564) ~[na:1.8.0]
	at java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1387) ~[na:1.8.0]
	at java.util.ResourceBundle.getBundle(ResourceBundle.java:845) ~[na:1.8.0]
	at org.thymeleaf.dialect.springdata.util.Messages.getMessage(Messages.java:17) ~[thymeleaf-spring-data-dialect-2.0.0.jar:na]
	at org.thymeleaf.dialect.springdata.util.Messages.getMessage(Messages.java:23) ~[thymeleaf-spring-data-dialect-2.0.0.jar:na]
	at org.thymeleaf.dialect.springdata.decorator.FullPaginationDecorator.getLaquo(FullPaginationDecorator.java:87) ~[thymeleaf-spring-data-dialect-2.0.0.jar:na]
	at org.thymeleaf.dialect.springdata.decorator.FullPaginationDecorator.decorate(FullPaginationDecorator.java:30) ~[thymeleaf-spring-data-dialect-2.0.0.jar:na]
	at org.thymeleaf.dialect.springdata.PaginationAttrProcessor.processAttribute(PaginationAttrProcessor.java:28) ~[thymeleaf-spring-data-dialect-2.0.0.jar:na]
	at org.thymeleaf.processor.attr.AbstractAttrProcessor.doProcess(AbstractAttrProcessor.java:87) ~[thymeleaf-2.1.4.RELEASE.jar:2.1.4.RELEASE]
	at org.thymeleaf.processor.AbstractProcessor.process(AbstractProcessor.java:212) ~[thymeleaf-2.1.4.RELEASE.jar:2.1.4.RELEASE]
	at org.thymeleaf.dom.Node.applyNextProcessor(Node.java:1017) ~[thymeleaf-2.1.4.RELEASE.jar:2.1.4.RELEASE]
…
´´´

I wonder if there should be a default properties file using only symbols and numbers which could be used in case no explicit translation has been provided?

**Comment by jpenren** [comment]
Some tags, like pagination summary, requires translation. At the moment, you can create your own locale files in your app, just create this files:
```
src/main/resources/thymeleaf-spring-data-dialect/PaginationSummary_de.properties
src/main/resources/thymeleaf-spring-data-dialect/PagerDecorator_de.properties
src/main/resources/thymeleaf-spring-data-dialect/FullPaginationDecorator_de.properties
src/main/resources/thymeleaf-spring-data-dialect/AlignedLinksDecorator_de.properties
```
See source code to know about parameter keys.

The next release will provide more language translations and default locale file.

**Comment by jpenren** [comment]
Fixed in 2.1.0 version
